jdaviz/core/validunits.py

```diff
@@ -73,12 +73,12 @@ def create_flux_equivalencies_list(flux_unit, spectral_axis_unit):
 
     # Get local flux units.
     locally_defined_flux_units = ['Jy', 'mJy', 'uJy', 'MJy',
-                                  'W / (Hz m2)',
-                                  'eV / (s m2 Hz)',
-                                  'erg / (s cm2 Hz)',
-                                  'erg / (s cm2 Angstrom)',
+                                  'W / (m2 Hz)', 'W / (Hz m2)',  # Order is different in astropy v5.3  # noqa
```

Comment:
Does the order matter? If so, do we need to include both to be compatible with different supported astropy versions?

Reply:
The duplicated units are now removed, maybe they should be there for changing units from the API (since the way that the angle_unit is appended for the UI is just at the end of the denominator and not the astropy location of the angle unit).